tools/fpgainfo/fmeinfo.c

```diff
@@ -56,7 +56,8 @@ static void print_fme_info(fpga_token token)
 	fpgainfo_board_info(token);
 	fpgainfo_print_common("//****** FME ******//", props);
 	fpga_boot_info(token);
-	fpga_image_info(token);
+	//TODO enable after fixing BMC log driver
```

Comment:
Just curious, when will the driver be fixed?

Reply:
N6010 Beta feature: Event log / reading NVM , not need for Agilex beta release.